python/tvm/topi/cuda/nms.py

```diff
@@ -554,12 +757,7 @@ def non_max_suppression(
     score_axis = score_index
     score_shape = (batch_size, num_anchors)
     score_tensor = te.compute(score_shape, lambda i, j: data[i, j, score_axis], tag=tag.ELEMWISE)
-    target = tvm.target.Target.current()
-    if (
-        target
-        and target.kind.name == "cuda"
-        and tvm.get_global_func("tvm.contrib.thrust.sort_nms", allow_missing=True)
-    ):
+    if tvm.get_global_func("tvm.contrib.thrust.sort_nms", allow_missing=True):
```

Comment:
Opencl doesn't have thrust lib, assert target is cuda here.

Reply:
Good catch, thanks!